assets/node-exporter/daemonset.yaml

```diff
@@ -23,7 +23,6 @@ spec:
         - --path.sysfs=/host/sys
         - --path.rootfs=/host/root
         - --no-collector.wifi
-        - --no-collector.hwmon
```

Comment:
This modifies the generate assets directly, you need to modify the jsonnet file to remove this, see this guide how to then generate https://github.com/openshift/cluster-monitoring-operator/blob/master/CONTRIBUTING.md

Reply:
Hmm, this was the only file in this repository that had that string in it. Where is this file coming from upstream?

Reply:
PR that removed `hwmon` from enabled collectors in upstream repository: https://github.com/prometheus-operator/kube-prometheus/pull/381